pkg/scanner/scanner.go

```diff
@@ -136,21 +142,39 @@ func (s *Scanner) listenForScanRequests() {
 				backendResults, err := backend.Scan(reqResource)
 				if err != nil {
 					logger.Error("scan error: request_id=%q error=%q", request.ID, err.Error())
+					request.Errors = append(request.Errors, LeakTKError{
+						Fatal:   true,
+						Code:    ScanError,
+						Message: err.Error(),
+					})
 				}
 				if backendResults != nil {
 					results = append(results, backendResults...)
 				}
 			}
 			if err := s.removeResourceFiles(reqResource); err != nil {
 				logger.Error("resource file cleanup error: request_id=%q error=%q", request.ID, err.Error())
+				request.Errors = append(request.Errors, LeakTKError{
+					Fatal:   false,
+					Code:    ResourceCleanupError,
+					Message: err.Error(),
+				})
 			}
 		} else {
 			logger.Error("skipping scan due to missing clone path: request_id=%q", request.ID)
+			request.Errors = append(request.Errors, LeakTKError{
+				Fatal:   true,
+				Code:    CloneError,
+				Message: fmt.Sprintf("missing clone path: request_id=%q (%s)", request.ID, reqResource.ClonePath()),
+			})
+		}
+		for _, e := range request.Errors {
+			fmt.Println(e)
 		}
```

Comment:
```suggestion
```

I don't think we want to include the errors in stdout here since they'll be passed along.